torch/distributed/distributed_c10d.py

```diff
@@ -1634,6 +1668,19 @@ def scatter_object_list(
         is known to be insecure. It is possible to construct malicious pickle
         data which will execute arbitrary code during unpickling. Only call this
         function with data you trust.
+
+    Example::
+        >>> # Note: Process group initialization omitted on each rank.
+        >>> import torch.distributed as dist
+        >>> if dist.get_rank() == 0:
+        >>>     objects = ["foo", 12, {1: 2}] # any picklable object
+        >>> else:
+        >>>     objects = [None, None, None]
```

Comment:
does this have to be 3 Nones?

Reply:
It's mostly to point out that whatever is contained in `objects` on non-src ranks, it doesn't matter, we will only look at the `objects` from the src rank. The 2nd arg can be anything for non-src ranks, it is not considered.